.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Open Pull Request
+
+on:
+  push:
+    branches:
+      - pulls/**
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: ./
+        with:
+          helper: create-pr
+          title: New PR
+          target: main
```

Comment:
We can default this value to `main` in action.yml

Reply:
I don't see any other default values in `action.yml`. What makes this the first candidate, and what is the format?

Reply:
Sorry, I commented below about the `target_branch` input. It may not be needed

Reply:
Also, the idea behind these workflows is that it provides an opportunity to test your helper on this repo!